software/controller/lib/sensors/analog_sensor.tpp

```diff
@@ -13,11 +13,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
```

Comment:
Maybe add warning not to include, similar to other tpp files ?

Reply:
Done.